common/stats/stats_names.go

```diff
@@ -18,6 +18,9 @@ const (
 	ClusterRunningNodes   = "runningNodes"
 	ClusterLostNodes      = "lostNodes"
 
+	ClusterNodeUpdateFreqMs = "clusterGetNodeUpdatesFreqMs"
```

Comment:
nit: existing stats_names convention is "_ms" vs "Ms"

Reply:
fixed